NEWS.md

```diff
@@ -1,251 +0,0 @@
-## DataFrames v0.6.11 Release Notes
```

Comment:
NEWS.md was removed in https://github.com/JuliaData/DataTables.jl/pull/1 since it didn't apply to DataTables, but now that we're merging back into DataFrames maybe we should keep it?